dist/origin-web-common-services.js

```diff
@@ -3193,6 +3194,9 @@ angular.module('openshiftCommonServices')
                                         });
                               }, function(e) {
                                 context.projectPromise.reject(e);
+                                if ((e.status === 403 || e.status === 404) && _.has(opts, 'skipErrorOnProjNotFound')) {
```

Comment:
`_.has` is wrong here since it will return true even when `skipErrorOnProjNotFound === false`. Use `_.get`.

Reply:
I'd probably go with the shorter `skipErrorNotFound`

Reply:
`opts.error.notFound`?

Reply:
Fixed to 'opts.skipErrorNotFound'